### PR TITLE
Add test with oldest dependencies to CI

### LINKED ISSUE #1153: Add CI-tests with minimum versions of main libraries
**Is your feature request related to a problem? Please describe.**
pyaerocom has tests for the latest 3 python versions, while the main libraries, e.g. pandas, xarray, numpy require the latest version. pyaerocom should not require the latest version at all times, but allow for about 3 years of backward compatibility. E.g. the currently latest supported python version (3.10), xarray 2022.04 and pandas 1.5 should currently be supported for this year.

This should be implemented before #1066 .
This is related to #1140 and #1141 .

**Describe the solution you would like to see**
pyaerocom should be able to work with slightly older versions of the main libraries, as well as the newest versions.

### PR DESCRIPTION
**Pull request by thorbjoernl**
## Related issue number

fix #1153

## Checklist

* [ ] Start with a draft-PR
* [ ] The PR title is a good summary of the changes
* [ ] PR is set to AeroTools and a tentative milestone
* [ ] Documentation reflects the changes where applicable
* [ ] Tests for the changes exist where applicable
* [ ] Tests pass locally
* [ ] Tests pass on CI
* [ ] At least 1 reviewer is selected
* [ ] Make PR ready to review

### CODE REVIEW

**Comment by heikoklein** [comment]
Please don't increase the number of CI-tests, max 4. I'd rather like the 3.10 test to be the 'old version' test, including the old dependencies.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -101,3 +101,40 @@ jobs:
...
+        sudo apt install libudunits2-dev libgeos-dev libproj-dev proj-data proj-bin
+
+    - name: Setup environment
+      run: |
+        pip install --upgrade pip
+        pip install pdm
+        pdm lock --strategy direct_minimal_versions -v
+        pdm install -v
+        pip install pytest
+        pip freeze
```
Comment (avaldebe) [inline .github/workflows/CI.yml:130]: can this logic be moved to the tox environment for python 3.10?
that way the same test can be run locally and there is no need to change the jobs on CI
